src/riak_core_vnode_worker.erl

```diff
@@ -37,12 +38,11 @@
         modstate :: any()
     }).
 
--spec behaviour_info(atom()) -> 'undefined' | [{atom(), arity()}].
-behaviour_info(callbacks) ->
-    [{init_worker,3},
-     {handle_work,3}];
-behaviour_info(_Other) ->
-    undefined.
+-callback init_worker(partition(), Args :: term(), Props :: [{atom(), term()}]) ->
```

Comment:
Should `Props` be a proplist with type `[atom() | {atom(), term()}]` or `[proplists:property()]`?